src/resolvers/projectResolver.test.ts

```diff
@@ -1695,12 +1809,10 @@ function activateProjectTestCases() {
     const firstUserAccessToken = await generateTestAccessToken(
       SEED_DATA.FIRST_USER.id,
     );
-    const project = await saveProjectDirectlyToDb(createProjectData());
-    const deactiveStatus = await ProjectStatus.findOne({
-      id: ProjStatus.deactive,
```

Comment:
It was good idea that you get `statusId` when creating test project, so I changed the codes in tests to use that instead of updating status after creating project